docs/library/xyz-protocol.md

```diff
@@ -0,0 +1,13 @@
+# XYZ Protocol
+
+The XYZ protocol describes how a client (often from within a browser) can access tiled imagery. It provides a pyramid of tiles at some zoom level so that it can efficiently be displayed in web browsers. 
+
+#### XYZ tiles have these attributes:
+
+* Tiles are 256 × 256 pixels.
+* Tiles use the Web Mercator coordinate reference system (EPSG:3857).
+* Tiles are available between zoom levels 0 and 18.
+* Tiles are rendered in the PNG format with an alpha channel for transparency.
+* Grid is a rectangle with 2\*z rows and 2\*z columns, where z is the zoom level.
+* Grid uses 0,0 as the top, left corner in the grid.
+Tiles are found at the path `z/x/y.png`, where `z` is the zoom level, and `x` and `y` are the positions in the tile grid.
```

Comment:
This is incorrect for our projects.
1. we do use 256x256 tiles for raster and rasterized vector, but 3dtiles in XYZ protocol does not have a resolution
2. Web mercator is not the only CRS, it can vary, depends how you produce it.
3. You have an assumption of a quad 1x1 tiling scheme, this is also incorrect for our project.

IMO, the source you have copied it from made many assumptions therefore it is irrelveant to us, you can rewrite this paragraph in a more generic way or remove details that are too specific for example -> instead of saying 256 x 256 you can say a `tileSizeInPixels x tileSizeInPixels` but make a disticition from tiled data and tiled images.